.github/workflows/release-native.yml

```diff
@@ -83,12 +80,59 @@ jobs:
         with:
           username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.DOCKER_PASSWORD }}
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
```

Comment:
what does this do?

Reply:
I believe this is needed for the `platforms` setting in `docker/build-push-action`